packages/Babylonian-Printbugger.package/BPPrintbuggerContext.class/instance/untrack..st

```diff
@@ -1,5 +1,7 @@
 state
 untrack: aBPExample
-	
+
 	self countAllOut: aBPExample traceValues.
-	aBPExample removeDependent: self.
\ No newline at end of file
+	aBPExample removeDependent: self.
+	
+	exampleToIgnoredAnnotations removeKey: aBPExample uniqueIdentifier ifAbsent: ["there is nothing to remove"]
```

Comment:
switching to annotation -> true/false could also mean just calling
`self unignoreAll: aBPexample annotations`

Reply:
Resolved due to consersation above on `BPSwimlane>>isTraceValueVisible:`. The mapping stays as it is.